lib/rules/component-tags-order.js

```diff
@@ -123,9 +162,38 @@ module.exports = {
               (e1, e2) => getOrderPosition(e1.name) - getOrderPosition(e2.name)
             )[0]
           if (firstUnordered) {
-            report(element, firstUnordered)
+            reportCandidates.push({ element, firstUnordered })
           }
         })
+        if (reportCandidates.length) {
+          const replaceRange = [
+            elements[0].range[0],
+            elements[elements.length - 1].range[1]
+          ]
+          const sortTagList = getSortedTagList(
+            elements.map((element, index) => ({
+              name: element.name,
+              index
+            }))
+          )
+          const sourceCode = context.getSourceCode().text
+          const reOrderedCode = Array.from(
+            { length: elements.length },
+            (_, i) => {
+              const [start, end] = elements[sortTagList[i].index].range
+              const code = sourceCode.slice(start, end)
+              return code
+            }
+          ).join('\n')
+          reportCandidates.forEach((error) => {
```

Comment:
@ota-meshi, i report each descriptor here, because i want to share the final souceFile, it could improve a little performance.

Reply:
> because i want to share the final souceFile, it could improve a little performance.

I think it's more important that suppressive comments work than performance. We need to change it to move only the reported element.